local/index.js

```diff
@@ -0,0 +1,43 @@
+const blockModule = volBlockStarterBlock
+
+const globalStyles = {
+  color: {},
+  globalComponents: {},
+  typography: {}
+};
+
+const props = { 
+  ...blockModule.defaultConfig,
+  text: 'Custom prop value for local testing'
+}
+
+function createBlock() {
+  const ElementPropTypes = window.ElementSdk.ElementPropTypes
+  const Components = window.ElementSdk.Components
+  const utils = window.ElementSdk.client
```

Comment:
This should be a an object with a `client` key. Here's the version of the utils object I like to use:

```js
        const utils = {
            client: window.ElementSdk.client,
            seo: window.ElementSdk.seo,
            isRendering: true,
            queryParams: mockQueryParams(),
            isAmpRequest: false,
            canonicalUrl: obj => {
                const keys = Object.keys(obj);
                const uriQuery = keys
                    .map((key, index) => {
                        const queryStart = index === 0 ? "?" : "&";
                        return queryStart + key + "=" + encodeURI(obj[key]);
                    })
                    .join("");
                return window.location.origin + uriQuery;
            },
            pubSub: {
                publish: (event, data) => {
                    const { productId, quantity, variantId } = data;
                    console.log(
                        `Product variant ID "${variantId}" should be added to cart`
                    );
                }
            },
            events: {
                cart: {
                    addToCart: () => {}
                }
            },
            throwNotFound: () => {
                console.log("Throw not found");
            }
```

Reply:
Although it looks like you could probably grab `events` and `PubSub` from `window.ElementSdk` too.

Reply:
For future reference: 
```
        var mockQueryParams = () => {
            const params = {};
            const searchParams = window.location.search;
            if (searchParams) {
                const urlParams = searchParams.replace("?", "").split("&");
                urlParams.forEach(param => {
                    const [key, value] = param.split("=");
                    params[key] = decodeURI(value);
                });
            }
            return params;
        };
```

Reply:
For now, I'm updating utils to the following:

```js
  const utils = {
    client: window.ElementSdk.client
  }
```
I'll open a new PR to import a more complete object from `element-block-utils` once it's been added there.